src/main/java/com/poiji/bind/Poiji.java

```diff
@@ -1,16 +1,18 @@
 package com.poiji.bind;
 
 import com.poiji.bind.mapping.UnmarshallerHelper;
-import com.poiji.exception.IllegalCastException;
-import com.poiji.exception.InvalidExcelFileExtension;
-import com.poiji.exception.PoijiExcelType;
-import com.poiji.exception.PoijiException;
+import com.poiji.exception.*;
 import com.poiji.option.PoijiOptions;
 import com.poiji.option.PoijiOptions.PoijiOptionsBuilder;
 import com.poiji.util.Files;
+import org.apache.poi.hssf.OldExcelFormatException;
+import org.apache.poi.hssf.extractor.OldExcelExtractor;
+import org.apache.poi.poifs.filesystem.DocumentFactoryHelper;
+import org.apache.poi.poifs.filesystem.FileMagic;
+import org.apache.poi.poifs.filesystem.NPOIFSFileSystem;
+import org.apache.poi.util.IOUtils;
 
-import java.io.File;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
could you change this statement with only the statement the file needs? for example: 

```
import java.io.File;
import java.io.InputStream;
```

Reply:
：） OK，i will fix it.it's really a good practice  improve to improve my code coverage.